Readme.md

```diff
@@ -234,6 +234,15 @@ I will follow my element movements
 </a>
 ```
 
+####Tooltip hidden
+I case you want to keep the tooltips hidden for some reasons, the `tooltip-hidden`argument can be set. In case you want to hide the tooltips and disable the `tooltip-show-trigger` the `tooltip-hidden` attribute must be assigned to `tooltip-hidden="true"`. To switch the tooltip behaviour dynamically, assign the `tooltip-hidden`attribute to a variable or function with type of boolean.
+Default: `false`
+```html
+<a href="#" tooltips tooltip-title="Turn me on or off" tooltip-hidden="{{tooltipHidden}}">
```

Comment:
Hi

I'm using these tooltips in my project. First of all thanks for sharing them. I'm really waiting to have the hide option. But I'm kindof new to github and don't know in which branch it will be available?

Javier